### DISCUSSION TITLE
Common.ExcelExportHelper error

### DISCUSSION THREAD

**Discussion by gfo2007** [serenity-is/Serenity#6754] [Help] [answered]
Hi,
I am unable to call ExcelExportHelper class in XXXgrid.ts to show the export button. It used to be under Common Module.

Below is the error.

![excel export error](https://user-images.githubusercontent.com/14317420/233831605-c1cb5e6f-063e-496f-8409-27f3ef11f56b.png)

**Comment by edwardch** [comment, marked as answer]
Have a look in the common-features repository, and under the Northwind module have a look at the OrderGrid.ts

https://github.com/serenity-is/common-features/blob/master/src/Serenity.Demo.Northwind/Modules/Order/OrderGrid.ts

**Reply by gfo2007** [reply]
@edwardch thank you. I got my answers from the link you provided.
